openipam/hosts/management/commands/bulk_delete_hosts.py

```diff
@@ -0,0 +1,83 @@
+from django.core.management.base import BaseCommand
+import sys
+import re
+import os
+
+from openipam.hosts.models import Host, User
+
+
+class Command(BaseCommand):
+    MAC_ADDR_REGEX = "^([0-9A-Fa-f]{2}[:-]){5}([0-9A-Fa-f]{2})$"
+
+    CYAN_ANSI_SEQ = "\033[96m"
+    RED_ANSI_SEQ = "\033[91m"
+    END_ANSI_SEQ = "\033[0m"
+
+    help = (
+        RED_ANSI_SEQ
+        + "EXAMPLE: echo '11:11:11:11:11:11 11:11:11:11:11:12,11:11:11:11:11:13' | python3 manage.py bulk_delete_hosts --user A01234567"
+        + END_ANSI_SEQ
+    )
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "-f",
+            "--file",
+            help="Path to file containing line-or-comma-or-space delimited hosts",
+        )
+        parser.add_argument(
+            "-u", "--user", help="User name to delete mac addresses with"
+        )
+        parser.add_argument(
+            "-uid", "--user-id", help="User id to delete mac addresses with"
+        )
+
+    def handle(self, *args, **options):
+        if not bool(options["user"]) != bool(options["user_id"]):
+            raise Exception("Must specify, exclusively, username or user id")
+
+        user = (
+            User.objects.get(username=options["user"])
```

Comment:
.get() will throw a Database Exception if no record exists....You will have to put it in a try block it instead use filter().first()